### PR TITLE
Update 025_tip_create_new_authority.md

### PR DESCRIPTION
**Pull request by Tonterias**
Update to the new changes.

### CODE REVIEW

File: tips/025_tip_create_new_authority.md [hunk 1]
```diff
@@ -34,29 +38,36 @@ Do not forget to include your new role in your authorities.csv:
...
-	        data: {
-	            authorities: ['ROLE_USER'],
-	            pageTitle: 'Messages'
-	        },
-	        canActivate: [UserRouteAccessService]
-	    }
-	];
+and/or in your Angular files: jhiHasAnyAuthority=[‘ROLE_ADMIN’. ‘ROLE_X’ ……] or even consider to use it in the routes:
+
+export const appuserRoute: Routes = [
```
Comment (ruddell) [inline tips/025_tip_create_new_authority.md:58]: This code section needs to be indented to put the code in a pre-formatted block

You can preview the changes by clicking on the page icon at the top ([screenshot](https://user-images.githubusercontent.com/4294623/80382880-d9cc9700-8870-11ea-8ffc-e193dc29347e.png)) or for the current changes: https://github.com/jhipster/jhipster.github.io/pull/949/files?short_path=a2064be#diff-a2064bef6385b464cd1f30572c8c881f

Comment (Tonterias) [reply tips/025_tip_create_new_authority.md:58]: Thanks Jon

**Comment by avdev4j** [comment]
closed in favor of https://github.com/jhipster/jhipster.github.io/pull/979

@Tonterias if you need there is missing, please reopen this PR.

**Comment by mplawley** [comment]
> #979

@avdev4j, thank you for reviewing and merging.

**Comment by avdev4j** [comment]
btw @mplawley and @Tonterias you did the same good job. 
I was not aware of that and I merged the @mplawley's or first. I can't merge the second one because of conflicts. 
For sure if you think we can use both, feel free to fix conflicts and I will review it as soon as possible.

**Pull request closed without merging**